pkg/network/cni/cni.go

```diff
@@ -185,11 +189,18 @@ func cniToIgniteResult(r *gocni.CNIResult) *network.Result {
 	return result
 }
 
-func (plugin *cniNetworkPlugin) RemoveContainerNetwork(containerID string) error {
-	if err := plugin.initialize(); err != nil {
+func (plugin *cniNetworkPlugin) RemoveContainerNetwork(containerID string) (err error) {
+	if err = plugin.initialize(); err != nil {
 		return err
 	}
 
+	cleanupErr := plugin.cleanupBridges(containerID)
+	if cleanupErr != nil {
+		defer util.DeferErr(&err, func() error {
```

Comment:
why do you defer the return error?

Reply:
We can still try to remove the network if the cleanup fails.